examples/basecoin/cmd/basecoind/main.go

```diff
@@ -1,27 +1,74 @@
 package main
 
 import (
+	"encoding/json"
 	"fmt"
 	"os"
 
+	"github.com/spf13/cobra"
+
+	"github.com/tendermint/tmlibs/cli"
 	dbm "github.com/tendermint/tmlibs/db"
 	"github.com/tendermint/tmlibs/log"
 
-	"github.com/cosmos/cosmos-sdk/baseapp"
 	"github.com/cosmos/cosmos-sdk/examples/basecoin/app"
+	"github.com/cosmos/cosmos-sdk/server"
+	"github.com/cosmos/cosmos-sdk/version"
+)
+
+// gaiadCmd is the entry point for this binary
+var (
+	gaiadCmd = &cobra.Command{
```

Comment:
this is basecoin

Reply:
What are you referring too? Yeah this is a basecoin CLI... not sure why we chose to name gaia... I guess ultimately basecoin actually becomes gaia?? gaia is just basecoin with staking.. should be hella simple - then the hub is just gaia with a genesis block really :)

Reply:
i wrote the gaia main and copied it over to basecoin, since i could not test gaiad commands without a working app.

forgot to rename variable, thanks